pages/social-inbox/activity-pub.mdx

```diff
@@ -0,0 +1,170 @@
+# Activity Pub
+
+The Social Inbox implements the [ActivityPub
+specification](https://www.w3.org/TR/activitypub/)'s dynamic parts,
+such as handling incoming notifications, follow requests, likes, boosts,
+and sending out notifications, as well as related specifications, like
+[Webfinger](https://webfinger.net/)
+
+## Inbox
+
+The [Inbox](https://www.w3.org/TR/activitypub/#inbox) receives
+activities from other actors and remote instances, such as Like,
+Announce, Follow, and replies in the form of Create and Update.
+
+Depending on your blocklist and allowlist, activities may be put on
+pause, accepted or rejected.
+
+The Inbox is a paginated collection that holds any activity received and
+stored, irrespective of their paused/approved status.  Only query the
+Inbox to check what activities are there.
+
+Rejected activities are currently not stored anywhere.
+
+Example: <https://social.distributed.press/v1/@distributed@distributed.press/inbox>
+
+## Outbox (notifications)
+
+To notify followers of your activities, send them as a POST request to
+your outbox.
+
+## Followers
+
+The [Followers](https://www.w3.org/TR/activitypub/#followers) collection
+contains a list of remote Actors that have sent a Follow request and was
+approved.
```

Comment:
Unauthenticated clients can request the followers collection and get back just the `totalItems` for rendering in user interfaces. The owner of the followers collection can make an authenticated request to get the full list.